### ISSUE TITLE
Can't enable sliding sync

### ISSUE THREAD

**Issue by lnicola** [vector-im/element-web#26504]
### Steps to reproduce

1. log in to a Conduit HS
2. try to enable sliding sync

#### What did you expect?

It works.

#### What happened instead?

 - the server responds after 30 seconds, but Element times out after 5 seconds (okay, maybe this is on the server)
 - if it didn't time out in the previous step, Element still seems to try to validate the proxy address by requesting `/client/server.json`, which Conduit doesn't know about

### Operating system

Linux

### Browser information

Firefox

### URL for webapp

1.11.47

### Homeserver

Conduit 0.7.0-alpha

### Will you send logs?

No

**Comment by weeman1337** [comment]
@lnicola  Thank you for opening an issue. Unfortunately there is not enough information there for me to be able to reproduce it. 

Can you send logs? Does it work with the Element X mobile clients?

**Comment by lnicola** [comment]
Yes, I can send some logs, but I think a screen recording would be more clear. Anyway, I'll try to patch Conduit to work around the timeout first.

**Comment by lnicola** [comment]
Buggy form validation:

![image](https://github.com/vector-im/element-web/assets/308347/cd3aa0c8-5208-427e-903b-de9109453248)

Logs uploaded.

**Comment by t3chguy** [comment]
It is expecting the server.json file from the proxy, right now it only supports the proxy https://github.com/matrix-org/sliding-sync/blob/a8fb52626110b8084e0cc6026a894ce3093eaefb/v3.go#L178

**Comment by lnicola** [comment]
I managed to enable it, it's currently making a lot of requests (reloading the history?).

The `client/server.json` request goes to the Element URL, so there's no need to modify the server, dropping a file in there is enough.

**Comment by lnicola** [comment]
Update: it's making a bajillion requests to `_matrix/client/unstable/org.matrix.msc3575/sync?pos=XXXXXX&timeout=20000`, where `pos` is increasing each time by 1 (it's around 385000 at the moment) and the server replies with pretty much the same thing to each of those.

I suspect it's indeed not working very well.

**Comment by girlbossceo** [comment]
The other issue is most likely due to https://github.com/matrix-org/sliding-sync/commit/e3b110e13b54dfad91bfae90b052c01478db9ed2 which I reported https://gitlab.com/famedly/conduit/-/issues/404
